tests/test_xml_translate.py

```diff
@@ -22,10 +22,10 @@ def setUp(self):
         self.not_a_file = str(current_dir / 'tests' / 'data' / 'not_a_file.xml')
         self.not_a_valid_xml = str(current_dir / 'tests' / 'data' / 'not_a_valid_xml.xml')
 
-    def test_xml2xml_1(self):
+    def test_xml_translate_1(self):
```

Comment:
instead of "1" - could you name it more explicitly in order to describe what you're testing?